src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java

```diff
@@ -230,23 +239,23 @@ public void vuapResolvedShouldEnd(Page page, String slot, String videoIframeSele
   @Test(
       dataProviderClass = AdsDataProvider.class,
       dataProvider = "adsVuapClickToPlayDesktop",
-      groups = "AdsVuapClickToPlaySizes"
+      groups = {"AdsVuapClickToPlaySizes"}
   )
-  public void vuapCheckSlotSizesOasis(Page page, String slot, String videoIframeSelector) {
-    AdsBaseObject ads  = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+  public void vuapCheckSlotSizesOasis(Page page, String slot) {
+    AdsBaseObject ads = new AdsBaseObject(driver, page.getUrl());
+    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, findFirstIframeWithAd(slot), false);
     scrollToSlot(slot, ads);
 
     double adSlotHeight = vuap.getAdSlotHeight();
-
-    vuap.playVuapVideo();
-
-    vuap.pause();
-
+    vuap.clickOnArea(2);
+    vuap.togglePause();
     double videoAdHeight = vuap.getVideoHeightWhilePaused();
 
-    VuapAssertions.verifyVideoAdSize(vuap, videoAdHeight, adSlotHeight,
-                                     MAX_AUTOPLAY_MOVIE_DURATION);
+    VuapAssertions.verifyVideoAdSize(vuap, videoAdHeight, adSlotHeight, MAX_AUTOPLAY_MOVIE_DURATION);
+  }
+
+  private By findFirstIframeWithAd(String slotName) {
```

Comment:
I'm not sure, but I have a feeling that this method should be moved to page object.